src/HealthChecks.EventStore.gRPC/EventStoreHealthCheck.cs

```diff
@@ -0,0 +1,38 @@
+using EventStore.Client;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace HealthChecks.EventStore.gRPC;
+
+public class EventStoreHealthCheck : IHealthCheck
+{
+    private readonly EventStoreClient _client;
+
+    public EventStoreHealthCheck(string connectionString)
+    {
+        ArgumentNullException.ThrowIfNull(connectionString);
+
+        _client = new EventStoreClient(EventStoreClientSettings.Create(connectionString));
+    }
+
+    /// <summary>
+    /// Checks whether a gRPC connection can be made to <see cref="EventStore"/>, using the supplied <c>connectionString</c>

```

Comment:
It's beter to move general description into xml for class level and here just use /// \<inheritdoc />

Reply:
Fixed 👍